app/controllers/websites_controller.rb

```diff
@@ -1,18 +1,16 @@
 class WebsitesController < ApplicationController
-
   def index
-    case params['sort']
-    when 'mobile_asc'
-      @websites = Website.mobile_asc
-    when 'mobile_desc'
-      @websites = Website.mobile_desc
-    when 'desktop_asc'
-      @websites = Website.desktop_asc
-    when 'desktop_desc'
-      @websites = Website.desktop_desc
-    else
-      @websites = Website.all
-    end
+    @websites = case params['sort']
+                when 'mobile_asc'
+                  Website.mobile_asc
+                when 'mobile_desc'
+                  Website.mobile_desc
+                when 'desktop_asc'
+                  Website.desktop_asc
+                when 'desktop_desc'
+                  Website.desktop_desc
+                else
+                  Website.all
```

Comment:
The 'all' isn't going to organize the records in a way that's meaningful to a human.  I think it'll just order them by their db id.  Can you make it default to ordering `Website.mobile_asc` by default?